code/modules/mob/living/silicon/robot/gripper.dm

```diff
@@ -11,39 +11,35 @@
 
 	flags = NOBLUDGEON
 
+	 //This is a list of items that can't be held even if their parent is whitelisted.
+	var/list/blacklist = list(
+		/obj/item/gun,
+		/obj/item/projectile,
+		/obj/item/storage,
+		/obj/item/modular_computer,
+		/obj/item/remains,
+		/obj/item/wheelchair,
+		/obj/item/stool,
+		/obj/item/shield
+		)
```

Comment:
Borgs are allowed to hold all obj/items currently.
Blacklist is used to restrict specific things, such as guns, storage, computers, stools, etc. 
No one wants a borg hitting you with a stool.